pages/events/[eventsId].js

```diff
@@ -32,6 +32,22 @@ export default function EventsDetail({ handleUpdate }) {
     handleUpdate(eventObject);
   };
 
+  // const handleDeleteEvent = () => {
+  //   const updatedEvents = events.filter((event) => event.id !== eventToEdit.id);
+  //   handleUpdate(updatedEvents);
+  //   setEventObject(null);
+  // };
```

Comment:
It's better not to leave code that is commented out. Remove it when you don't use it.